Q3.js

```diff
@@ -0,0 +1,15 @@
+//In the following shopping cart add, remove, edit items
+
+const shoppingCart = ['Milk', 'Coffee', 'Tea', 'Honey']
+// add 'Meat' in the beginning of your shopping cart if it has not been already added
+// add Sugar at the end of you shopping cart if it has not been already added
+// remove 'Honey' if you are allergic to honey
+// modify Tea to 'Green Tea'
+shoppingCart.unshift('meat');
```

Comment:
@Fakhara77 it says add `Meat` not `meat` if it is not already added.
Why you are no checking if it is already added.

Reply:
resolved